src/vs/workbench/browser/parts/titlebar/titlebarPart.ts

```diff
@@ -713,19 +710,19 @@ export class BrowserTitlebarPart extends Part implements ITitlebarPart {
 		return !this.isAuxiliary && this.configurationService.getValue(LayoutSettings.ACTIVITY_BAR_LOCATION) === ActivityBarPosition.TOP;
 	}
 
-	protected get useCounterZoom(): boolean {
+	get hasNoElementsToZoom(): boolean {
```

Comment:
I think this would be easier to read if we were not using negation. Why not call this `hasZoomableElements`